src/wallet.js

```diff
@@ -52,11 +58,19 @@ function WalletContextProvider({ children }) {
       balance: new BN(filterBalanceValue(balance)),
       ethereum,
       networkType,
-      providerInfo: getProvider(identifyProvider(ethereum)),
+      providerInfo: getProviderFromUseWalletId(activated),
```

Comment:
Should we do `activated || activating`?

Reply:
Hmm for safety I'd say yes, even though `activating` becomes `null` as soon as `activated` gets the id.

Reply:
Yes, this is so we can get `providerInfo` a little bit sooner :-)

Reply:
I'll commit, but can you test it? I think it produces an animation issue.

Reply:
It works fine here, what kind of issue do you have?

<img src="https://user-images.githubusercontent.com/36158/76142975-ba806d00-606a-11ea-8c45-4798a3564062.gif" width=400>

Reply:
Ahh, on my side it's not presenting the loading screen correctly, but I think that's on my end then. Merging!